CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Changed
+- Added in colour new variables for discovery and live badges and banners
+- Updated existing link colour variables to match gov.uk [#795](https://github.com/hmrc/assets-frontend/pull/795)
```

Comment:
I still think these should be merged into one point. Both colour updates are a single PR.

Reply:
Did you say that before? I may have missed that, I squash them into 1 point now.

Reply:
I did 😄 Thanks!